### DISCUSSION TITLE
{{ Screen.ReferredBy }} Binding

### DISCUSSION THREAD

**Discussion by poirazis** [Budibase/budibase#7674] [Ideas]
The scenario : 
we have a generic entity, i.e meeting, that can be created / attached / related to different entites, for example a working_group, a department can have one or more meetings. So, when we hit "New meeting" from the department page, or "New meeting" from the working_group page, we want the /meeting/new/row screen to adjust some parameters. In order to achieve this, we either pass the initiating entity as a route parameter, for example /meeting/new/row/:parent_type/:parent_id or we could expose a {{ Screen.ReferrredBy }} binding, that will bring forth the URL that called this screen so we can extract data and offer a navigation back to parent after saving.

Maybe is can be part of the {{ Query Params }} binding ?

Thoughts ?

**Comment by melohagan** [comment]
This might be a related issue here: https://github.com/Budibase/budibase/issues/5420
It would be good to handle some way!
